server/routes/positions.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const express = require('express');
+const logger = require('../util/logger');
+
+const router = express.Router();
+const Position = require('../models/Position');
+
+router.get('/search', async (_req, res) => {
```

Comment:
Change _req to req'. "_req" is the naming convention to use to signify we are not using anything in the request.

Reply:
ah thanks I didn't know that, good to know :)